content/extractor.js

```diff
@@ -16,36 +16,122 @@
 
 // Given DOM element returns array of possible id-links to resolve.
 
+import { flatten } from 'lodash';
+
 export const TAGS_BLACKLIST = [ 'script' ];
 
-export function extractPossibleMatches ($dom) {
-  const tag = $dom.tagName.toLowerCase();
+const EMAIL_PATTERN = /([^\s@]+@[^\s@]+\.[a-z]+)/i;
```

Comment:
This regex does not conform to the email address RFC. It's common sense not to use regexes to validate email addresses. However, because this extension *has to* do some sort of matching, I'd recommend to keep `@[^\s@]+\.[^\s@]+` in the end.